solr/contrib/scripting/src/test/org/apache/solr/scripting/handler/XsltUpdateRequestHandlerTest.java

```diff
@@ -69,8 +77,8 @@ public void testUpdate() throws Exception
       "</random>";
 
     Map<String,String> args = new HashMap<>();
-    args.put(CommonParams.TR, "xsl-update-handler-test.xsl");
-      
+    args.put(XSLTParams.TR, "xsl-update-handler-test.xsl");
```

Comment:
In tests, you can just refer to the parameter directly: "tr".  Yonik advised this practice once on the reasoning that the test catches API changes you might not have meant to make.  Also, it's shorter :-)